### DISCUSSION TITLE
Log Forwarding

### DISCUSSION THREAD

**Discussion by eliassteiner** [okd-project/okd#2062] [Q&A]
Hello everyone,
I have the following question: How do I forward logs from an OKD cluster to an external ELK or Splunk, for example? Licensing-wise, it might be difficult to use the Red Hat OpenShift Logging Operator, right? (That's how I understood the documentation?)
I have now found this operator: https://github.com/redhat-openshift-ecosystem/community-operators-prod/tree/main/operators/logging-operator. But is it also from OpenShift? The next solution would be https://kube-logging.dev/docs/install/.
Is there a good explanation of what the best practice is, or which solution is also acceptable to use?

Thanks for the help!

**Comment by sitzma** [comment]
to forward logs to Splunk we use https://github.com/signalfx/splunk-otel-collector

**Comment by SupremeMortal** [comment]
Using the RedHat operator catalog with OKD does violate the license, however we have a work-in-progress operator catalog that has an OKD built version of the Logging operator which is identical to the Red Hat variant without any of the branding. https://github.com/okd-project/okderators-catalog-index

**Reply by LukaszBisewski** [reply]
Hi, thank you for sharing this update regarding the OKD operator catalog. Could you clarify:
Is there an estimated timeline for when this work-in-progress OKD operator catalog will be fully available for production use?
Are there any alternative solutions currently recommended for logging in OKD while we wait for this catalog to be finalized?

Thanks!
